.github/workflows/container-image-buildah.yml

```diff
@@ -0,0 +1,100 @@
+name: Container Image
+
+on:
+  workflow_dispatch: {}
+  workflow_call: {}
+  schedule:
+    # every Wednesday morning
+    - cron: 7 7 * * 3
+  push:
+    branches: [ main ]
+    tags:
+      - '*'           # Push events to every tag not containing /
```

Comment:
an image will be built and pushed to ghcr.io for all tags that do not contain `/` in their name

Reply:
Will existing, manually pushed images to gchr.io conflict with those?

Reply:
If you build from a tag/branch that already has an image pushed in whatever way, the newly image will override the old image.

If you ask about the existing 2 images, unless you change the existing tags or trigger a build manually, then no new builds will automatically be created.

Does this answer your question?